[STT-7] fix(ui): Update UI due to changes with Planning RelatedEvents schema

MarkLark86 opened the pull request.

dependabot[bot] committed:
Bump coverage from 7.5.1 to 7.5.3 (#1994)

Bumps [coverage](https://github.com/nedbat/coveragepy) from 7.5.1 to 7.5.3.
- [Release notes](https://github.com/nedbat/coveragepy/releases)
- [Changelog](https://github.com/nedbat/coveragepy/blob/master/CHANGES.rst)
- [Commits](https://github.com/nedbat/coveragepy/compare/7.5.1...7.5.3)

---
updated-dependencies:
- dependency-name: coverage
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

MarkLark86 committed:
Update interfaces

MarkLark86 committed:
Update redux reducers

MarkLark86 committed:
Update redux selectors

MarkLark86 committed:
Update redux actions

MarkLark86 committed:
Update websocket notification handling

MarkLark86 committed:
Update internal PlanningAPI

MarkLark86 committed:
Update UI components

MarkLark86 committed:
Update utils

MarkLark86 committed:
Update package-lock.json

MarkLark86 committed:
fix unit test

tomaskikutis commented on client/interfaces.ts:
```diff
@@ -694,14 +694,22 @@ export interface IPlanningCoverageItem {
     scheduled_updates: Array<ICoverageScheduledUpdate>;
 }
 
+export type IPlanningRelatedEventLinkType = 'primary' | 'secondary';
```
could you add a comment describing what primary/secondary means?

tomaskikutis commented on client/actions/planning/notifications.ts:
```diff
@@ -301,13 +302,20 @@ const udpateAssignment = (planningId) => (
             }
         });
+
+        return Promise.all(promiseResponses);
```
`promises` would be a better name. Promise "response" I would understand as a resolved value.

MarkLark86 committed:
Add comment to IPlanningRelatedEventLinkType interface, update variable name

petrjasek committed:
merging `release/2.7` to develop (#1998)

* [SDBELGA-778] fix: Show recurring Planning update method on Ignore|Cancel|Save modal (#1930)

* [SDBELGA-778] fix: Show recurring Planning update method on Ignore|Cancel|Save modal
Also improve look and feel when rendering Event & Planning metadata in the modal

* fix(ci): Use superdesk/release/2.7 branch for test instance

* fix extension build error

* add new should_udpate func [SDCP-742] (#1963)

* add new should_udpate func [SDCP-742]

* update params

* refactore logic

* Onclusive parse new key for event status mapping [SDCP-749] (#1970)

* Onclusive parse new key for event status mapping [SDCP-749]

* address comment

* Fix content profile closing after canceling a groups' creation (#1964)

* implement onclusive reingesting (#1971)

SDCP-751

* drop angular-history (#1933)

* drop angular-history

* fix test build

* sync redux versions with client core

* update package-lock

* update events during ingest if name/subject changes

SDCP-751

* allow removing contact email/phone via update (#1979)

SDCP-762

* set expiry when parsing onclusive events (#1995)

* set expiry when parsing onclusive events

CPCN-825

* fix mypy

* bump version

* update ui framework version (#1997)

* fix ci

* Set version to 2.8.0-dev

---------

Co-authored-by: MarkLark86 <mark.pittaway@gmail.com>
Co-authored-by: Ketan <73937490+devketanpro@users.noreply.github.com>
Co-authored-by: Konstantin Markov <maga.ot.oz@protonmail.com>

MarkLark86 committed:
Merge branch 'develop' into STT-7
